canonical-web-and-design/ubuntu.com
# Editable defaults cleanup 345

## Done

- Editable defaults cleanup 
  - add EntitlementLabel type
  - extract feature form utils to `FeaturesTab/utils.ts`
  - add unit/integration tests
- add cypress tests

## QA
- Go to /advantage
- Make sure features can be toggled on and off and preferences saved

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/345
